src/doom/g_game.c

```diff
@@ -1197,7 +1224,8 @@ void G_Ticker (void)
 
 	    memcpy(cmd, &netcmds[i], sizeof(ticcmd_t));
 
-	    if (demoplayback) 
+        // [crispy] don't read input from demo when paused
+	    if (demoplayback && !paused)
```

Comment:
I don't think this paused check should be here. We already checked `paused & 2` above this, and otherwise we have a pause inside of the demo, from which we need to continue reading tics.